admin/views/main/tmpl/default.php

```diff
@@ -161,7 +161,14 @@
 							<td><b><?php echo $this->category->total; ?> </b></td>
 						</tr>
 					</table>
-					<?php echo JHtml::_('tabs.end'); ?>
+					<?php echo JHtml::_('sliders.end'); ?>
+					<?php echo JHtml::_('sliders.panel', JText::_('COM_JEM_MAIN_DONATE')); ?>
```

Comment:
This will cause an error because second parameter (the panel id) is missing. Also a panel outsite the slider doesn't make really sense.
Please let me accept this PR and fix it in our repo...